index.php

```diff
@@ -2,6 +2,19 @@
 require_once './functions.php';
 $db = new PDO('mysql:host=db; dbname=kantopokemon', 'root', 'password');
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+if(isset($_POST['pokedex_id']) && isset($_POST['name']) && isset($_POST['type1']) && isset($_POST['type2']) && isset($_POST['region'])) {
+    $otherQuery = $db->prepare('INSERT INTO `kanto-pokemon` (`pokedex_id`, `name`, `type_1`, `type_2`, `hp`, `region`) VALUES (:pokedexId, :name, :type1, :type2, :hp, :region);');
+
+    $pokedexId = $_POST['pokedex_id'];
+    $name = $_POST['name'];
+    $type1 = $_POST['type1'];
+    $type2 = $_POST['type2'];
+    $hp = $_POST['hp'];
+    $region = $_POST['region'];
```

Comment:
need to validate these fields:
- do any of them need to be nonempty? name and pokedex id probably, maybe others too
- hp needs to be an int

so you need to check that the input values are valid according to those rules before inserting into the db